docs/openapi/components/schemas/credentials/CommercialInvoiceCredential.yml

```diff
@@ -668,11 +668,11 @@ properties:
                         enum:
                           - QuantitativeValue
                     value:
-                      title: Value
+                      title: Weight Value
```

Comment:
Why Weight Value/ Weight Unit? I thought this was supposed to be generic to allow invoices of many types.

Reply:
We want to be able to render documents without having to include the full hierarchy. So this is to distinguish for example Gross Weight from Net Weight in a flattened rendering. 

Note this is only the schema title - the key is still the generic `value`, so it's still wired into the generic class and its term definitions.